05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,39 @@
-.lawn {
-    width: 100%;
-    height: 300px;
-}
+/*CSS for Berries*/
 
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
+.image {
+  width:500px;
+  float:left;
 }
 
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+.image img {
+  width:100%;
+  height:100%;
 }
 
-.left {
-    float: left;
-    width: 60%;
-    height: 300px;
-    margin-left: 260px;
-    background-color: lightgray;
+.text {
+  width:500px;
+  height:398px;
+  float:right;
+  background-color: rgb(240,240,240);
+  text-align: center;
 }
 
-.left h1 {
-    text-align: center;
-}
+a {
+  font-weight: bold;
```

Comment:
font-weight needs numeric values. "bold" values start at 700 and are in- and decrimented in 100's.